bridge_adaptivity/module/migrations/0007_collection_updated_at.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
In my opinion we should create abstract model with fields ```created_at``` and ```updated_at``` and inherit mostly all models from this model. 

It will give us ability to know when objects were created and updated.

Reply:
We will think about it to figure out which models require timestamp fields. And if find that it would be helpful will add them in one of the following PR.